src/packages/@ncigdc/components/ActionsRow.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { compose } from 'recompose';
+
+import { stringifyJSONParam } from '@ncigdc/utils/uri';
+
+import { Row } from '@ncigdc/uikit/Flex';
+import Button from '@ncigdc/uikit/Button';
+import withRouter from '@ncigdc/utils/withRouter';
+import AnnotationsLink from '@ncigdc/components/Links/AnnotationsLink';
+import { CreateRepositoryCaseSetButton } from '@ncigdc/modern_components/withSetAction';
+import { fetchFilesAndAdd } from '@ncigdc/dux/cart';
+import { ShoppingCartIcon } from '@ncigdc/theme/icons';
+import DownloadManifestButton from '@ncigdc/components/DownloadManifestButton';
+
+export default compose(
+  connect(),
+  withRouter,
+)(({ filters, totalCases, dispatch, totalFiles, push }) => {
```

Comment:
can you type this so it's clear `totalCases` and `totalFiles` are numbers and not arrays of objects.

```
import type { TGroupFilter } from '@ncigdc/utils/filters/types';

[...]
)(({ filters, totalCases, dispatch, totalFiles, push }: {filters: TGroupFilter, totalCases: number, totalFiles: number, dispatch: Function, push: Function}) => {
[...]
```
you get the idea